tests/nested/manual/core20-install-device-file-install-ubuntu-save-via-hook/task.yaml

```diff
@@ -0,0 +1,143 @@
+summary: Verify that ubuntu-save is available during install-device hook with specific device
+
+details: |
+    This test checks that the workaround put in place for a specific device
+    continues to work. This specific device needs to install configuration files
+    onto the run system ubuntu-data on UC20 during install mode to avoid an
+    additional reboot during the first boot process.
+
+systems: [ubuntu-20.04-64, ubuntu-22.04-64]
+
+environment:
+    # use snapd from the spread run so that we have testkeys trusted in the
+    # snapd run
+    NESTED_BUILD_SNAPD_FROM_CURRENT: true
+
+    # use secure boot and TPM to enable encryption
+    NESTED_ENABLE_TPM: true
+    NESTED_ENABLE_SECURE_BOOT: true
+
+    # don't use cloud-init it doesn't work with grade secured
+    NESTED_USE_CLOUD_INIT: false
+
+    # sign all the snaps we build for the image with fakestore
+    NESTED_SIGN_SNAPS_FAKESTORE: true
+
+    # use the testrootorg auto-import assertion
+    NESTED_CUSTOM_AUTO_IMPORT_ASSERTION: $TESTSLIB/assertions/developer1-{VERSION}-auto-import.assert
+    NESTED_CUSTOM_MODEL: $TESTSLIB/assertions/developer1-{VERSION}-secured.model
+
+    # for the fake store
+    NESTED_FAKESTORE_BLOB_DIR: $(pwd)/fake-store-blobdir
+    NESTED_UBUNTU_IMAGE_SNAPPY_FORCE_SAS_URL: http://localhost:11028
+
+    # unset this otherwise ubuntu-image complains about overriding the channel for
+    # a model with grade higher than dangerous when building the image
+    NESTED_CORE_CHANNEL: ""
+
+prepare: |
+    if [ "$TRUST_TEST_KEYS" = "false" ]; then
+        echo "This test needs test keys to be trusted"
+        exit
+    fi
+
+    # install pre-reqs which we need to adjust various bits
+    snap install jq remarshal
+    tests.cleanup defer snap remove remarshal
+    tests.cleanup defer snap remove jq
+
+    #shellcheck source=tests/lib/store.sh
+    . "$TESTSLIB"/store.sh
+
+    # setup the fakestore, but don't use it for our snapd here on the host VM, so
+    # tear down the staging_store immediately afterwards so that only the SAS is 
+    # running and our snapd is not pointed at it, ubuntu-image is the only thing 
+    # that actually needs to use the fakestore, and we will manually point it at
+    # the fakestore below using NESTED_UBUNTU_IMAGE_SNAPPY_FORCE_SAS_URL
```

Comment:
I was about to say that this is very hard to read (and I still think it is!), and I don't understand what it says, but then I noticed that this comment has been copied from other tests.

If you understand what this comment says, please try to reword it (and break it down into periods :-) ), otherwise I guess it's fine to leave it here as it is.

Reply:
I tried to reword it a bit, I agree it was less than ideally worded. I hope it seems a bit more readable now

Reply:
Oh, yes, it's much more clear now, thanks! :-)